app/controllers/Application.scala

```diff
@@ -99,16 +99,18 @@ class Application(
     Ok(views.html.unsupportedBrowserPage())
   }
 
-  def contributionsLanding(countryCode: String, maybeSSR: Option[String]): Action[AnyContent] = maybeAuthenticatedAction().async { implicit request =>
+  def contributionsLanding(countryCode: String, maybeSSR: Option[String], maybeFormDesignTest: Option[String]): Action[AnyContent] = maybeAuthenticatedAction().async
```

Comment:
The build is failing because some of the lines here are > 160 characters, so you need to try and break them down over a couple of lines